.github/workflows/test-and-upload-coverage.yml

```diff
@@ -28,6 +28,7 @@ jobs:
     name: Run tests matrix job
 
     strategy:
+      fail-fast: true
```

Comment:
suggestion: making it false would let other runners complete even if one runner is flakey and fails. Otherwise all will fail upon one flakey runners and would need to re-run all failed runners instead of just the flakey one.

Reply:
I think such a discussion/change is out of scope here, and does not need to interrupt the delivery of this PR.

Reply:
The addition of this line within this PR does affect / change the behavior of the ci.

Reply:
Only if github changes it's default.  I will remove this line if you feel like this is a change, although I did bring it into this PR after you suggested I do so with this `true` value (in another PR).

Reply:
Okay let's keep it for some reason I thought I suggested the other way around. (I wanted to maintain the default github behavior, which seems like this maintains).